src/System.Security.Cryptography.Cng/src/ApiCompatBaseLine.net463.txt

```diff
@@ -0,0 +1,9 @@
+MembersMustExist : Member 'System.Security.Cryptography.CngAlgorithm.ECDiffieHellman.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
Added `# The .NET 4.6.3 targeting pack (1.0.1) doesn't have the new members added for ECDsa import/export yet.`